po/Makefile.am

```diff
@@ -114,22 +114,20 @@ update-po: po/cockpit.pot
 		$(MSGMERGE) --output-file=$(srcdir)/po/$$lang.po $(srcdir)/po/$$lang.po $<; \
 	done
 
-ZANATA_CLI = LANG=C.UTF-8 LC_ALL=C.UTF-8 zanata-cli -B
+ZANATA_CLI = LANG=C.UTF-8 LC_ALL=C.UTF-8 zanata
 
 upload-pot: po/cockpit.pot
-	$(ZANATA_CLI) push --includes cockpit.pot --project-config "$(srcdir)/po/zanata.xml" \
-		-s "$(builddir)/po"
+	$(ZANATA_CLI) push -f --srcdir=$(builddir)/po --push-type=source --project-config=$(srcdir)/po/zanata.xml cockpit.pot
 
 upload-translations:
-	$(ZANATA_CLI) push --includes=cockpit.pot --project-config "$(srcdir)/po/zanata.xml" \
-		--push-type trans -s "$(builddir)/po" -t "$(srcdir)/po"
+	$(ZANATA_CLI) push -f --srcfile=cockpit.pot --srcdir=$(srcdir)/po --push-type=both \
+		--srcdir=$(builddir)/po --transdir=$(srcdir)/po --project-config=$(srcdir)/po/zanata.xml
```

Comment:
I just realized that I haven't tested this command. @martinpitt do we use it for something?

Reply:
Indeed it seems we don't -- I would have thought po-refresh would do that, so that we get a two-way sync between git and zanata. But it seems we only upload the pot. I guess because we never actually update translations in git? But this can just be tested manually, it should be a no-op.

Reply:
What does `--push-type=both` mean? It sounds like "all po plus pot"? If it's that, then this rule should depend on po/cockpit.pot. But as we already have `upload-pot`, it seems better to me to have a `--push-type=only-the-po-files` and handle pot separately.